nixos/modules/services/networking/networkmanager.nix

```diff
@@ -177,7 +179,7 @@ in {
       basePackages = mkOption {
         type = types.attrsOf types.package;
         default = { inherit (pkgs)
-                            networkmanager modemmanager wpa_supplicant crda
```

Comment:
Every reference to `networking.networkmanager.basePackages` should drop `wpa_supplicant`.
(gnome3, etc.)

Reply:
Hm, why were so many places re-setting the basePackages? Just trying to understand why/what the change is...

Reply:
It appears to be a hack so networkmanager can use the gnome version of the packages from the `gnome3` attrset. GNOME type libs are pretty popular so, they're built with those features.

Reply:
services.xserver.desktopManager.xfce4-14 as well